react/src/components/Table/Table.tsx

```diff
@@ -364,6 +317,18 @@ const Table: React.FC<TableProps> = ({ variantData }) => {
                         width: 79,
                     },
                     { accessor: 'transcript', id: 'transcript', Header: 'transcript', width: 150 },
+                    {
+                        accessor: 'homozygousCount',
+                        id: 'homozygousCount',
+                        Header: 'homozygous count',
+                        width: 150,
+                    },
+                    {
+                        accessor: 'heterozygousCount',
+                        id: 'heterozygousCount',
+                        Header: 'heterozygous count',
+                        width: 150,
```

Comment:
You might want to use `getColumnWidth` here. The function calculates the content to adjust width accordingly, versus hardcoding the width.